plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/StandardColumnMappings.java

```diff
@@ -489,6 +521,15 @@ public static LongWriteFunction timestampWriteFunction(TimestampType timestampTy
         return (statement, index, value) -> statement.setObject(index, fromPrestoTimestamp(value));
     }
 
+    public static ObjectWriteFunction longTimestampWriteFunction(TimestampType timestampType)
+    {
+        checkArgument(timestampType.getPrecision() > MAX_SHORT_PRECISION && timestampType.getPrecision() < MAX_LOCALDATETIME_PRECISION,
+                "Precision is out of range: %s", timestampType.getPrecision());
```

Comment:
fmt - we put all arguments on one line, or each on separate line

Reply:
Done!